assets/css/main.scss

```diff
@@ -0,0 +1,13 @@
+---
+# assets/main.scss
+---
+// set your variables or @import them here.
+
+// Setup for the US Web Design Standards
+$font-path: '../uswds/fonts';
+$image-path: '../uswds/img';
+@import 'uswds/all';
+
+
+// styles for buttons in navigation
```

Comment:
Wondering if there is a better way to incorporate this `nav-buttons` CSS optionally. Thinking about this for other components that might need some default styling.

Reply:
I think for something like this, it's lightweight enough that we can probably just include it, but maybe for larger components we could set a config option and conditionally include it into the main.scss file.